scripts/fast-reboot

```diff
@@ -791,6 +791,7 @@ for service in ${SERVICES_TO_STOP}; do
             sonic-db-cli ASIC_DB FLUSHDB > /dev/null
             sonic-db-cli COUNTERS_DB FLUSHDB > /dev/null
             sonic-db-cli FLEX_COUNTER_DB FLUSHDB > /dev/null
+            sonic-db-cli RESTAPI_DB FLUSHDB > /dev/null
```

Comment:
This will get cleared for NVIDIA's warmboot (fastfast-reboot). I think that is not expected.

Should this check be restricted to `fast-reboot`? @prsunny ?

Reply:
I updated the code. Now it's limited to `fast-reboot` only.